doc/cascadia/profiles.schema.json

```diff
@@ -684,8 +684,20 @@
         },
         "useTabSwitcher": {
           "default": true,
-          "description": "When set to \"true\", the \"nextTab\" and \"prevTab\" commands will use the tab switcher UI.",
-          "type": "boolean"
+          "description": "When set to \"true\" or \"mru\", the \"nextTab\" and \"prevTab\" commands will use the tab switcher UI, with most-recently-used ordering. When set to \"inOrder\", the tab switcher will switch tabs in their current ordering. Set to \"false\" to disable the tab switcher.",
```

Comment:
Took me a minute here. So the UI is used for either true, MRU, or inOrder. And false is no UI. The way it's described its sort of like "true"/"mru" = UI and MRU, "inorder" = ???? and in order, "false" = No UI. Like it's weirdly implicit that the UI is used for in order and explicit for the other two options. 

Or maybe I'm just crazy.